cmd/common/registry.go

```diff
@@ -400,7 +409,34 @@ func buildCommandFromGadget(
 			formatter := parser.GetTextColumnsFormatter()
 
 			requestedStandardColumns := outputModeParams == ""
-			requestedColumns := strings.Split(outputModeParams, ",")
+			requestedColumns := make([]string, 0)
+
+			// Check, if columns were requested relatively
+			// (using only +column and -column syntax)
+			addCols := make([]string, 0)
+			removeCols := make([]string, 0)
+			requestedRelativeColumns := true
```

Comment:
super minor naming nit: `requestedAllRelativeColumns`, otherwise I would think this is true if a single relative column was requested.

Reply:
Just in time :D I was almost doing `git push ...`